queue_classic_admin.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib,vendor}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_runtime_dependency "rails", ">= 5.0.0", "< 7.1"
```

Comment:
Maybe we can set smaller supported version to Rails 6.1?.

Also the failed unit tests indicate:
```
ArgumentError: The `legacy_connection_handling` setter was deprecated in 7.0 and removed in 7.1, but is still defined in your configuration. Please remove this call as it no longer has any effect."
```
which is defined in `application.rb`, so we can probably get rid of that line.

Reply:
@sebaherrera07 yep, that works - we should only support current versions; I think 6.1.x is the oldest still being patched.